src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -14,36 +14,33 @@ describe('RuleInformationProvider', () => {
         provider = new RuleInformationProvider();
     });
 
-    function buildTouchSizeWcagRuleResultObject(
-        status: string,
-        dpi: number,
-        height: number,
-        width: number,
-        axeViewId: string = null,
```

Comment:
axeViewId should have been removed when RuleInformationProvider was factored out. Fixing the oversight now.